src/contrib/cluster/Akka.Cluster.Tools/PublishSubscribe/DistributedPubSubMediator.cs

```diff
@@ -131,13 +131,13 @@ public static Props Props(DistributedPubSubSettings settings)
         /// <summary>
         /// TBD
         /// </summary>
-        public IDictionary<Address, long> OwnVersions
+        public IImmutableDictionary<Address, long> OwnVersions
```

Comment:
Technically a breaking API change but not really - no one uses the actor's API directly.